packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -14,19 +14,29 @@ import uuid from 'uuid';
 import {InvalidUserCreation} from './convo-error';
 import ShareActivity from './share-activity';
 
+const idToUrl = new Map();
 
 const Conversation = WebexPlugin.extend({
   namespace: 'Conversation',
 
+  /**
+   * @param {Object} conversation
+   * @param {Object} object
+   * @param {Object} activity
+   * @returns {Promise}
+   */
   acknowledge(conversation, object, activity) {
+    const url = this.getConvoUrl(conversation);
+    const space = Object.assign({}, conversation, {url});
```

Comment:
The `space` var name is a bit jarring to me, personally. It doesn't seem to fit with the `conversation` plugin. Perhaps something like `conversationWithUrl` would be more readable.